src/utils/constants.ts

```diff
@@ -51,13 +51,78 @@ const CONFIGURE_TABS: ConfigureTab[] = [
   { type: 'wallet', label: 'Wallet' },
 ]
 
+type SocialLogin = {
+  name: 'Google' | 'Twitter' | 'Twitch' | 'Reddit' | 'Github' | 'Discord'
+  verifier: 'google' | 'twitter' | 'twitch' | 'reddit' | 'github' | 'discord'
+  hasClientSecret: boolean
+  hasRedirectUri: boolean
+  documentation: string
+  clientId?: string
+  clientSecret?: string
+  redirectUri?: string
+}
+
+type SocialAuth = {
+  verifier: string
+  clientId?: string
+  clientSecret?: string
+  redirectUri?: string
+}
+
+const socialLogins: SocialLogin[] = [
+  {
+    name: 'Google',
+    verifier: 'google',
+    hasClientSecret: false,
+    hasRedirectUri: false,
+    documentation: 'https://developers.google.com/identity/sign-in/web/sign-in',
+  },
+  {
+    name: 'Reddit',
+    verifier: 'reddit',
+    hasClientSecret: false,
+    hasRedirectUri: false,
+    documentation: 'https://github.com/reddit-archive/reddit/wiki/OAuth2',
+  },
```

Comment:
We can skip Reddit for now until [AR-2127](https://team-1624093970686.atlassian.net/browse/AR-2127) is resolved.